network-api/networkapi/wagtailpages/templates/wagtailpages/campaign_page.html

```diff
@@ -8,91 +8,95 @@
 {% endcomment %}
 
 {% block subcontent %}
-<div class="
-  {% block content_wrapper_class %}py-4 py-md-5{% endblock %}
-  {% if page.specific.narrowed_page_content %}
-    col-lg-8 offset-lg-2
-  {% else %}
-    col-12
-  {% endif %}"
->
+<div class="container">
```

Comment:
since we removed the container and row divs from modular page to enable full-width content on blog pages, we are re-adding them here in campaign _page and some other files as well to ensure they work like before.

I have also made this change in the image-block PR, but since I was not sure which branch would get merged in first, I added it to this one as well